docs/docs/ref-01-top-level-api.md

```diff
@@ -110,6 +110,15 @@ boolean isValidElement(* object)
 Verifies the object is a ReactElement.
 
 
+### React.findDOMNode
+
+```javascript
+DOMElement findDOMNode(ReactComponent component)
+```
+
+Finds the native DOM element, if any, associated with the given ReactComponent.
```

Comment:
Can you just copy the text from here? http://facebook.github.io/react/docs/component-api.html#getdomnode

Maybe worth adding a note there too saying that React.findDOMNode is preferred.